src/dataflow/core/program_utils.py

```diff
@@ -194,6 +199,15 @@ def mk_call_op(name: str, args: List[Idx], idx: Idx = 0) -> Tuple[Expression, Id
     return flat_exp, new_idx
 
 
+def mk_type_name(sexp: Sexp) -> str:
```

Comment:
docstring

Reply:
This one makes more sense in #37 so I'll fix it there.